JEasyCryptoClient/src/CryptoClient.java

```diff
@@ -74,7 +74,9 @@ public void run() {
 			} while (choice != QUIT_MENU);
 			
 		} catch (SocketException e) {
-			e.printStackTrace();
+			System.out.println("Error: The socket is not reached.");
```

Comment:
Also I think we should be more specific, since `SocketException` is very general and covers more than a non reachable socket. See [here](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/SocketException.html) for reference.

Could you please provide exceptions and `System.err.println` for the children? This would give the user much more context.